sqe-http-api/DTOs/User.cs

```diff
@@ -20,10 +20,11 @@ public class UserDTO
 
     public class LoginResponseDTO : UserDTO
     {
-        public string token { get; set; }
+        public string token { get; set; } // Used with successful login to activated account
```

Comment:
Isn't this really the *only place* where we return a token? And nowhere else? So we don't really need a token in the UserDTO - just all the fields?